libs/ledgerjs/packages/cryptoassets/src/abandonseed.ts

```diff
@@ -17,8 +17,8 @@ const abandonSeedAddresses: Partial<Record<CryptoCurrency["id"], string>> = {
   ripple: "rHsMGQEkVNJmpGWs8XUBoTBiAAbwxZN5v3",
   stellar: "GDYPMQMYW2JTLPWAUAHIDY3E4VHP5SGTFC5SMA45L7ZPOTHWQ2PHEW3E",
   tezos: "tz1VJitLYB31fEC82efFkLRU4AQUH9QgH3q6",
-  // https://developers.tron.network/reference/getburntrx-1
-  tron: "TLsV52sRDL79HXGGm9yzwKibb6BeruhUzy",
+  // Randomly generated inactive tron address.
```

Comment:
I think it was nice to know how it was generated. The link seems to still be relevant